package.json

```diff
@@ -56,6 +56,56 @@
     "onStartupFinished"
   ],
   "main": "./out/extension.js",
+  "runme": {
+    "features": [
+      {
+        "name": "ForceLogin",
+        "enable": true,
+        "conditions": {
+          "os": "All",
+          "vsCodeVersion": ">=1.58.0",
+          "allowedExtensions": [
```

Comment:
We could also turn `allowedExtensions` into `enabledForExtensions` and then make the keys the extension handle and its value a true or false boolean. Something like this:

```
      "gist": {
        "enabled": true,
        "conditions": {
          "os": "All",
          "vsCodeVersion": ">=1.58.0",
          "enabledForExtensions": {
            "stateful.runme": true,
            "stateful.platform": false
          }
        }
      }
```

Again, the goal being that overwrites are an exception and not part of packaging the extension.